README.md

```diff
@@ -70,7 +70,7 @@ Tested in Node 8+.
 Create a Cosmiconfig explorer, then either `search` for or directly `load` a configuration file.
 
 ```js
-const cosmiconfig = require('cosmiconfig');
+const { cosmiconfig } = require('cosmiconfig');
```

Comment:
Within the PR let's document this breaking change under the heading `HEAD` in `CHANGELOG.md`.

Reply:
Added!